### DISCUSSION TITLE
Error al entregar código, Lab 2.03- Pandas

### DISCUSSION THREAD

**Discussion by ddavidSM** [rramosp/ai4eng.v1#75] [02 - PYTHON]
Estoy teniendo problemas a la hora de entregar los códigos a la plataforma de autocorrección del módulo 2 de python, concretamente el laboratorio de pandas, no me asigna una nota al entregar el código sino que saca un mensaje de error. Cabe destacar que en colab el código funciona adecuadamente al correrlo, y al revisar manualmente encuentro que el código es correcto. Adjunto una captura del error que saca el sistema, quedo a la espera de su respuesta, gracias.

![imagen](https://user-images.githubusercontent.com/95448251/144508762-6cafe33d-c55e-4e5d-b33f-4d2321245bbf.png)

**Comment by Felipe-RA** [comment]
Saludos ddaviSM

El error indica que no encontró una variable **d**  a la hora de extrar los datos con _tu función select_items_ .

Para poder encontrar específicamente cuál es tu problema es necesario ver tu código. Puedes compartir tu notebook o al menos un snippet de tu función _select_items_
